api_specifications/hearing-api-0.1.0.yaml

```diff
@@ -255,6 +269,11 @@ components:
         tenantId:
           type: string
           description: This is tenantId of the case
+        hearingId:
+          type: string
+          description: The formatted number given when a hearing gets created.
+          minLength: 2
```

Comment:
from what I understand it is still a UUID. There is no hearing number like we have CNR For Case or Order ID for order. This is internal ID only

Reply:
This is generated via idgen , want me to remove it?